org.compiere.db.oracle.provider/src/org/compiere/db/DB_Oracle.java

```diff
@@ -570,288 +567,64 @@ public String[] getCommands (int cmdType)
         return null;
     }   //  getCommands
 
-    private String getFileName ()
+	private String getPoolPropertiesFile ()
 	{
-		//
 		String base = null;
-		if (Ini.isClient())

```

Comment:
we have deprecated the swing client and I think it is time to gradually remove all this Ini.isClient() check from source.

Reply:
I've removed the `if (Ini.isClient())` in both the PostgreSQL and Oracle DB code.

Reply:
What needs to remain is the Ini.isClient() == false part (i.e the else part here).

Reply:
Right, I need more coffee apparantly ;-). Fixed!